### DISCUSSION TITLE
Acrylic background for Microsoft Sticky Notes

### DISCUSSION THREAD

**Discussion by emvaized** [Maplespe/DWMBlurGlass#194] [Ideas]
I use [Microsoft Sticky Notes](https://apps.microsoft.com/detail/9NBLGGH4QGHW) app to constantly show "notepad-like" sticky note on the desktop. 

Would it be possible somehow to inject styles into this app and remove the background, making it use Acrylic effect as a background? Similar to how it is done in your another wonderful app [ExplorerBlurMica](https://github.com/Maplespe/ExplorerBlurMica)

**Comment by Maplespe** [comment]
There was a similar idea in the previous [ExplorerBlurMica discussion](https://github.com/Maplespe/ExplorerBlurMica/issues/97).
There I think you will have the answer.
But it doesn't apply to any of the existing projects, and we might consider writing a separate new project to implement it in the future.

**Reply by emvaized** [reply]
Oh, that's wonderful! Looks exactly like what I'm looking for. But I'm not sure I'll be able to at least register on the website to download this 😅

**Reply by ALTaleX531** [reply]
You seem to have misunderstood, we meant that we might consider creating a separate project to give Microsoft Sticky Notes acrylic backdrop, which we have not implemented yet. ~~As this is commercial software and its rendering procedures are not handled by Dwm, I personally think it is unlikely that the feature can be implemented.~~

Edit: It's quite possible to implement it.

**Reply by emvaized** [reply]
So, this suggested toolbox currently can not modify Sticky Notes windows, and for now there's no sense for me to try installing it to achieve it?

**Reply by ALTaleX531** [reply]
Yes, please wait for further updates.

**Reply by emvaized** [reply]
Okay, got it. Thanks! I want this feature so much.

**Reply by emvaized** [reply]
Hi there! Any updates on this?

**Reply by ALTaleX531** [reply]
We don't have enough time to start a new project, and it will likely never be implemented.
